documentation/release_notes.markdown

```diff
@@ -1,9 +1,31 @@
 ---
-title: "PuppetDB 2.2 » Release Notes"
+title: "PuppetDB 2.3 » Release Notes"
```

Comment:
wouldn't bother doing this now ... this will be done across the board later anyway.